### ISSUE TITLE
Change delimiter for escaped strings

### ISSUE THREAD

**Issue by dloscutoff** [dloscutoff/pip#24]
After having `\"escaped strings\"` in the language for a while, I'm starting to think it may be better to use an unescaped quote character for the closing delimiter, and escape quotes inside the string instead:

    Old: \"double "quotes" unescaped\"
    New: \"double \"quotes\" escaped"

Situations for using escaped strings:

- There's a double quote in the string. If there is only one double quote, the change breaks even; if there are multiple double quotes, the change increases the byte-count by one for each double quote beyond the first one.
- Variable interpolation. For this use, the change would save one byte every time.

So the question is, which is more common--variable interpolation, or strings with multiple double quotes in them? Right now, the answer seems to be "neither"... quotes in string literals are rare, but variable interpolation isn't used much because there's usually an equally short way to do things:

    \"Hello, \y.\"
    "Hello, 0."R0y
    "Hello, ".y.'.

Shaving a byte off the escaped-string syntax would make interpolation more attractive:

    \"Hello, \y."
    "Hello, 0."R0y
    "Hello, ".y.'.

The situation is even better with multiple variables:

    \"I like \a, \b, and spam."
    "I like ".a.k.b.", and spam."
    "I like 0, 1, and spam."R,2[ab]

Meanwhile, most escaped strings containing double quotes already have an equally short alternative (ignoring precedence):

    \"This string contains "quotes"\"
    ST`This string contains "quotes"`

In summary, I don't think there's much of importance to lose, and perhaps something to gain. But more deliberation is in order before making a final decision.

**Comment by dloscutoff** [comment]
After further reflection, I've decided not to change anything.

- Using `\"` as the open delimiter but not the close delimiter, and then using the same `\"` inside the string for a literal quotation mark, is really confusing to look at.
- I can probably count on one hand the times I've actually used escaped strings in golf. There's no reason to spend effort changing something that won't get used anyway.
- While the examples above are shorter with the new syntax, they are fairly contrived and don't reflect the kind of output most code-golf challenges ask for.
